File: yongkeun/3week/369게임.js
function solution(order) {
  var answer = 0;
  var order = String(order).split(""); //29423을 2 9 4 2 3 으로 출력하기 위해 split을 이용해 분리
  order.forEach((value) => {
    if (value == 3 || value == 6 || value == 9) {
      //들어온 값이 3이거나 6이거나 9면
      answer++; //answer 증가
    }
  });
  return answer;
}

File: minwook/3주차/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        while (order != 0) {
            if (order % 10 == 3 || order % 10 == 6 || order % 10 == 9) 
            answer++; // 자릿수 별로 나눈값이 3 6 9 일 경우 값 추가
            order /= 10; // 그리고 자릿수 제거
        }
        return answer;
    }
}